src/ad-hoc-visualizations/needs-review/visualization.tsx

```diff
@@ -28,15 +31,25 @@ export const NeedsReviewAdHocVisualization: VisualizationConfiguration = {
     launchPanelDisplayOrder: 6,
     adhocToolsPanelDisplayOrder: 6,
     getAnalyzer: provider =>
-        provider.createBaseAnalyzer({
+        provider.createRuleAnalyzerUnifiedScanForNeedsReview({
+            rules: [
+                'aria-input-field-name',
+                'color-contrast',
+                'td-headers-attr',
+                'th-has-data-cells',
+            ],
+            resultProcessor: (scanner: ScannerUtils) => scanner.getFailingInstances,
+            telemetryProcessor: (telemetryFactory: TelemetryDataFactory) =>
+                telemetryFactory.forIssuesAnalyzerScan,
```

Comment:
We should probably have a function that refers to NeedsReview here instead of Issues for the telemetry.

Reply:
got it, added in telemetry specific to needs review